wp-includes/components/class-wp-service-worker-configuration-component.php

```diff
@@ -72,19 +72,24 @@ public function get_script() {
 		);
 		$script .= sprintf( "workbox.setConfig( %s );\n", wp_service_worker_json_encode( $options ) );
 
+		// Vary the prefix by the root directory of the site to ensure multisite subdirectory installs don't pollute each other's caches.
+		$prefix = sprintf(
+			'wp-%s',
+			wp_parse_url( WP_Service_Workers::SCOPE_FRONT === $current_scope ? home_url( '/' ) : site_url( '/' ), PHP_URL_PATH )
+		);
+
 		$cache_name_details = array(
-			// Vary the prefix by the root directory of the site to ensure multisite subdirectory installs don't pollute each other's caches.
-			'prefix'   => sprintf(
-				'wp-%s',
-				wp_parse_url( WP_Service_Workers::SCOPE_FRONT === $current_scope ? home_url( '/' ) : site_url( '/' ), PHP_URL_PATH )
-			),
+			'prefix'   => $prefix,
 			// Also precache name by scope (front vs admin) so that different assets can be precached in each respective application.
 			'precache' => sprintf( 'precache-%s', WP_Service_Workers::SCOPE_FRONT === $current_scope ? 'front' : 'admin' ),
 			'suffix'   => 'v1',
 		);
 
 		$script .= sprintf( "workbox.core.setCacheNameDetails( %s );\n", wp_service_worker_json_encode( $cache_name_details ) );
 
+		// Export the prefix as a constant since it is not accessible via workbox.core.cacheNames, unfortunately (yet).
+		$script .= sprintf( 'const WP_SERVICE_WORKER_CACHE_PREFIX = %s;', wp_service_worker_json_encode( $prefix ) );
```

Comment:
See https://github.com/GoogleChrome/workbox/issues/1998 for an enhancement to Workbox to allow accessing the cache prefix.

Reply:
And see fix in https://github.com/GoogleChrome/workbox/pull/2001 which we can take advantage of once the next version of Workbox is released.

Reply:
If we merge https://github.com/xwp/pwa-wp/pull/150 first then we can avoid adding this `const`.

Reply:
As Workbox 4.2.0 has been released with the `prefix` getter.